styleguide/source/assets/js/form-items.js

```diff
@@ -100,22 +100,6 @@
         })
         .trigger('checkval');
 
-        $('select')
-        .on('change', function()
-        {
-          var $this = $(this);
-
-          if ($this.val() == '')
-          $this.addClass('watermark');
-
-          else
-          $this.removeClass('watermark');
-
-          $this.trigger('checkval');
```

Comment:
@nstriedinger I think just this line needs to stay. `checkval` is a process that triggers the validation styles.